connector.go

```diff
@@ -77,6 +79,54 @@ type EntityInfo struct {
 	TTL *time.Duration
 }
 
+// StringSet is a set of strings.
+type StringSet map[string]struct{}
+
+// ScopeMetadata is metadata about a scope. (JSON tags to support MD setting CLI tools.) The scope
+// may be qualified by a prefix, as in "production.vsoffers".
+type ScopeMetadata struct {
+	Entity      `dosa:"primaryKey=(Name)" json:"-"`
+	Name        string     `json:"name"`
+	Owner       string     `json:"owner"` // Owning group name (or the same as Creator)
+	Type        int32      `json:"type"`  // Production, Staging, or Development
+	Flags       int64      `json:"flags"`
+	Version     int32      `json:"version"`
+	PrefixStr   string     `json:"prefix_str,omitempty"` // With ":" separators
+	Cluster     string     `json:"cluster,omitempty"`    // Host DB cluster
```

Comment:
I am not sure about he Cluster. What's the scenario that if service migrates between dosa clusters.

Reply:
Migration is TBD -- in the normal case, a scope lives on exactly one cluster. For migration we'll have to set up some additional machinery which will have "Source" and "Destination" clusters.